index.bs

```diff
@@ -803,9 +774,30 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
               value in {{MediaCapabilitiesEncodingInfo/configuration}}.
             </li>
             <li>
-              If the user agent is able to encode the media represented by
-              <var>configuration</var>, set {{MediaCapabilitiesInfo/supported}}
-              to <code>true</code>. Otherwise set it to <code>false</code>.
+              If <code>configuration.video</code> [=map/exists=], let
+              |videoMimeSupport| be the result of running [$Check MIME Type
+              Support$] with <code>configuration.video.\[[mimeType]]</code>,
+              the |media| video, and |configuration|'s
+              {{MediaEncodingConfiguration/type}}, otherwise
+              let |videoMimeSupport| be <code>undefined</code>.
+            </li>
+            <li>
+              If <code>configuration.audio</code> [=map/exists=], let
+              |audioMimeSupport| be the result of running [$Check MIME Type
+              Support$] with <code>configuration.video.\[[mimeType]]</code>,
+              the |media| video, and |configuration|'s
+              {{MediaEncodingConfiguration/type}}, otherwise
+              let |audioMimeSupport| be <code>undefined</code>.
```

Comment:
As above: ` <code>configuration.audio</code>`  should link to the component parts. Also, is configuration a map? ... no, it's a MediaEncodingConfiguration. 

```suggestion
              If "audio" [=map/exists=] in |configuration|, let
              |audioMimeSupport| be the result of running [=Check MIME Type
              Support=] with `configuration.video.\[[mimeType]]`,
              the |media| video, and |configuration|'s
              {{MediaEncodingConfiguration/type}}, otherwise
              let |audioMimeSupport| be <code>undefined</code>.
```